hack/generate-yamls.sh

```diff
@@ -67,7 +67,7 @@ readonly MONITORING_LOG_ELASTICSEARCH_YAML=${YAML_OUTPUT_DIR}/monitoring-logs-el
 
 declare -A CONSOLIDATED_ARTIFACTS
 CONSOLIDATED_ARTIFACTS=(
-  ["${MONITORING_YAML}"]="${MONITORING_LOG_ELASTICSEARCH_YAML} ${MONITORING_METRIC_PROMETHEUS_YAML} ${MONITORING_TRACE_ZIPKIN_YAML}"
+  ["${MONITORING_YAML}"]="config/monitoring/100-namespace.yaml ${MONITORING_LOG_ELASTICSEARCH_YAML} ${MONITORING_METRIC_PROMETHEUS_YAML} ${MONITORING_TRACE_ZIPKIN_YAML}"
```

Comment:
I think it's better to add https://github.com/knative/serving/pull/8512/files#diff-9b02a2cff65b39c15538d028609d4df7L144-L146 back but delete line 145, so

ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/100-namespace.yaml \
    | "${LABEL_YAML_CMD[@]}" > "${MONITORING_YAML}"